### PR TITLE
Add basic bonus round functionality

### PR DESCRIPTION
**Pull request by shannonmoranetz**
Implementing previously discussed strategy for creating a bonus round, fixing several fringe cases (ie. bankruptcy, losing a turn). 

@dylhof @robbiejaeger @brittanystoroz @thatPamIAm

### CODE REVIEW

File: lib/Game.js [hunk 1]
```diff
@@ -21,14 +21,18 @@ class Game {
...
-    })
+    });
   };
 
   checkDelimiters(character, positionNum) {
     if (character === '-') {
       $(`.letter-card-div-${positionNum}`).children().removeClass('hidden');
       $(`.letter-card-div-${positionNum}`).css({"background-color": "#27A067"});
     }
+    if (character === '\'') {
```
Comment (dylhof) [inline lib/Game.js]: This is a good conditional for the apostrophes.   We will need to try and test it to make sure it's working with the slash in there. If not maybe interpolation?

File: lib/Game.js [hunk 2]
```diff
@@ -21,14 +21,18 @@ class Game {
...
+      $(`.letter-card-div-${positionNum}`).children().removeClass('hidden');
+      $(`.letter-card-div-${positionNum}`).css({"background-color": "#27A067"});
```
Comment (dylhof) [inline lib/Game.js]: This is great! a good solution to solve the problem of our left over puzzle tiles looking like they are on the board.

File: lib/Game.js (hunk 1 above)
Comment (shannonmoranetz) [reply lib/Game.js]: Sounds good! Issue created regarding apostrophes, but likely priority P3 for now.

File: lib/Game.js [hunk 3]
```diff
@@ -107,28 +111,62 @@ class Game {
...
+            currentGame.checkLetterAnswer(character);
+            skipConsonant = !skipConsonant;
+          }
+        }
+      })
+    })
+  }
+
+  showBonusOptions() {
+    console.log('fire')
```
Comment (dylhof) [inline lib/Game.js]: We will need to make sure to get rid of all of our console logs.

**Pull request merged**